### DISCUSSION TITLE
I just wanted to ask if it's possible to add the background image

### DISCUSSION THREAD

**Discussion by crownzzZZ** [Lissy93/dashy#1158] [General]
![image](https://user-images.githubusercontent.com/124179860/230535804-31667462-0d97-4dc3-b787-de44b7a02d48.png)

Although I think it looks pretty good now, it would be great if I could add a background image, such as an anime image as the background image.If possible, can someone teach me how to do it?

Another point I would like to ask is why the exchange rate icon, the national flag, cannot be displayed.

**Comment by CrazyWolf13** [comment]
Surely 
![image](https://github.com/Lissy93/dashy/assets/96661824/7c058176-a46a-4228-a043-61bd6bef3bdd)
![image](https://github.com/Lissy93/dashy/assets/96661824/456660bc-4ae8-4d84-a387-2f384ae22b4c)
Application-Config at the bottom
then scoll down a bit and you will see background image.
Depending on if you have bound /public/icons to a folder locally you can just copy the the background image to there and reboot your container otherwise you have to use sudo docker cp {local-path} {container-name(with filename)}:{container-path(without filename)}
![image](https://github.com/Lissy93/dashy/assets/96661824/7c81cdf8-8040-4ae8-aa47-87e605be1c2e)

**Comment by insomnia417** [comment]
跟你同样的问题，我试出来了，你如果使用本地图片，他默认的文件夹路径是在/app/public，所以如果你要使用跟其他的小图标的路径下，你在backgroud image选项下，就要使用/item-icons/xxxx.jpg 之类的方式
